internal/cli/atlas/backup/snapshots/create_test.go

```diff
@@ -34,9 +34,7 @@ func TestCreateOpts_Run(t *testing.T) {
 
 	t.Run("diskBackupReplicaSet run", func(t *testing.T) {
 		expected := &atlasv2.DiskBackupSnapshot{
-			DiskBackupReplicaSet: &atlasv2.DiskBackupReplicaSet{
```

Comment:
Some of the oneOfs contained only single object which made SDK overly verbose and introduced unneded nested structure